stingray/lightcurve.py

```diff
@@ -194,6 +202,22 @@ def __init__(self, time, counts, err=None, input_counts=True,
                                               self.tstart + self.tseg]]))
         check_gtis(self.gti)
 
+    def change_mjdref(self, new_mjdref):
```

Comment:
This method does not change the state of the object internally, it returns a light curve!
The reason for this method is that I forced other methods to only consider two distinct light curves if they have the same MJDREF. For example, to calculate a cross spectrum we take two lightcurves. We want to make sure that they have the same time reference (otherwise, the times in seconds might not even overlap), and the easiest way to account for this was to make both light curves referred to the same MJDref. Another way might be using astropy Time objects directly, but it requires far more reworking of the code